src/main/java/ma/internetshop/controller/AllUsersController.java

```diff
@@ -0,0 +1,26 @@
+package ma.internetshop.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import ma.internetshop.lib.Injector;
+import ma.internetshop.model.User;
+import ma.internetshop.service.UserService;
+
+public class AllUsersController extends HttpServlet {
```

Comment:
let's add verbs to Controller's names for example  GetAllUsersController (here and everywhere)